### PR TITLE
Update decent exposure to 3.0

### PR DESCRIPTION
**Pull request by AlexeyKuznetzov**
Update to next major version of **Decent Exposure** .
Mild API changes: https://github.com/hashrocket/decent_exposure/wiki/Api-changes-in-version-3

**Notable changes**:
* **Decent Exposure** was updated to next major version (*3.0*)
* **Decent Decoraton** was updated to next minor version that support new *Decent Exposure* version (*0.1.0*)
* **Decent Exposure Strategy** was removed from *ApplicationController*
* **Controller scaffold** was updated according to new *Decent Exposure* API
* **Flash messages** in *controller scaffold* were removed

### CODE REVIEW

**Comment by slavakisel** [comment]
@timurvafin Please review

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -6,8 +6,4 @@ class ApplicationController < ActionController::Base
 
   responders :flash
   respond_to :html
-
-  decent_configuration do
```
Comment (AlexeyKuznetzov) [inline app/controllers/application_controller.rb]: According to https://github.com/hashrocket/decent_exposure/wiki/Api-changes-in-version-3#strategies:

> If you have set in your ApplicationController to work with strong parameters you can just remove this code:
> ```ruby
# v 2.x
class ApplicationController < ActionController::Base
  decent_configuration do
    strategy DecentExposure::StrongParametersStrategy
  end
end
```

> And you should implement explicitly your #{exposed_thing}_params method like:
> ```ruby
# v 3.x
class ThingController < ApplicationController
  def thing_params
    params.require(:thing).permit(:foo, :bar)
  end
end
```

File: lib/templates/rails/scaffold_controller/controller.rb [hunk 2]
```diff
@@ -2,16 +2,19 @@
 class <%= controller_class_name %>Controller < ApplicationController
   respond_to :html
 
-  expose(:<%= singular_table_name %>, attributes: :<%= singular_table_name %>_params)
-  expose(:<%= plural_table_name %>) { <%= class_name %>.page(params[:page]) }
+  expose :<%= singular_table_name %>
+  expose :<%= plural_table_name %>, -> { <%= class_name %>.page(params[:page]) }
 
   def create
     flash[:notice] = '<%= human_name %> was successfully created.' if <%= singular_table_name %>.save
```
Comment (timurvafin) [inline lib/templates/rails/scaffold_controller/controller.rb]: Since we use flash responder no need to define flash message here.

Comment (AlexeyKuznetzov) [inline lib/templates/rails/scaffold_controller/controller.rb]: Agree. I have removed them in the latest commit.

**Comment by timurvafin** [comment]
@AlexeyKuznetzov  could you please sync your branch with `master`?

**Comment by AlexeyKuznetzov** [comment]
@timurvafin Sure, it's synced now.

**Pull request merged**